wpeverest/user-registration
# Fix - Token mismatch

### All Submissions:


### Changes proposed in this Pull Request:

Previously, when we clicked on the confirmation email link, it redirected to my account page. If we try to log in with the wrong password then it shows the wrong password as well as a token mismatch error. This PR fixes that issue.

### How to test the changes in this Pull Request:

1. Go to any registration form -> change its login options to auto approval after email confirmation
2. Then try to register a new user and click on the email verification link
3. Try to log in with the wrong password. and verify token mismatch error is coming or not

### Types of changes:


### Other information:


### Changelog entry

> Fix - Token mismatch.
